pkg/controller/plan/migration.go

```diff
@@ -871,7 +894,11 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			break
 		}
 
-		ready, err := r.builder.ConvertPVCs(pvcs)
+		srcFormatFn := func(pvc *core.PersistentVolumeClaim) string {
+			return pvc.Annotations[base.AnnSourceFormat]
```

Comment:
so if we say that for export-OVA (@liranr23 @bkhizgiy ) we won't need this, why having this generalization rather than taking the source format from the annotation within the converter?

Reply:
we can, but it feels weird in terms of API usability, it makes the format hidden that way

Reply:
I wouldn't say it's hidden that way, but yeah - it relies on the annotation that we rely on also this way..

Reply:
ok, a bit of an over-spec here but it's not too complex..